assets/email_template.txt

```diff
@@ -4,15 +4,15 @@
   |\\ | |__  __ /  ` /  \\ |__) |__         }  {
   | \\| |       \\__, \\__/ |  \\ |___     \\`-._,-`-,
                                         `._,._,'
-  nf-core/vcftomaf v${version}
```

Comment:
should probably also remove the logo. Did you check how the nf-core create without branding command handles that?

Reply:
You can remove the [whole logo](https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/assets/email_template.txt#L1-L10) :)